src/cformat.jl

```diff
@@ -3,7 +3,7 @@ formatters = Dict{ String, Function }()
 sprintf1( fmt::String, x ) = eval(Expr(:call, generate_formatter( fmt ), x))
 
 function checkfmt(fmt)
-    test = Base.Printf.parse( fmt )
+    test = @static VERSION >= v"1.4.0-DEV" ? Printf.parse(fmt) : Base.Printf.parse( fmt )
```

Comment:
Could you update the version test so that it will work with both older and newer 1.4.0-DEV.
```
... VERSION >= v"1.4.0-DEV.180" ...
```

Reply:
Done!